build/archives/BUILD.gn

```diff
@@ -29,21 +35,31 @@ zip_bundle("artifacts") {
       destination = "frontend_server.dart.snapshot"
     },
   ]
+
+  exe = ""
   if (host_os == "win") {
-    files += [
-      {
-        source = "$root_out_dir/flutter_tester.exe"
-        destination = "flutter_tester"
-      },
-    ]
-  } else {
-    files += [
-      {
-        source = "$root_out_dir/flutter_tester"
-        destination = "flutter_tester"
-      },
-    ]
+    exe = ".exe"
   }
+  dll = ".so"
+  if (host_os == "win") {
+    dll = ".dll"
+  } else if (host_os == "mac") {
+    dll = ".dylib"
+  }
+  files += [
+    {
+      source = "$root_out_dir/flutter_tester$exe"
+      destination = "flutter_tester$exe"
+    },
+    {
+      source = "$root_out_dir/impellerc$exe"
+      destination = "impellerc$exe"
+    },
+    {
+      source = "$root_out_dir/libtessellator$dll"
+      destination = "libtessellator$dll"
```

Comment:
Note for future us: we did something a little strange here and manually prepended `lib` to the DLL output to avoid a naming clash on Windows with a source_set that otherwise had the same name in Impeller. This has the happy (?) side effect of making the file name without the extension being identical on all platforms.